app/src/main/java/com/orange/ods/app/ui/components/navigationdrawers/ComponentModalDrawers.kt

```diff
@@ -0,0 +1,168 @@
+                            trailing = OdsSwitchTrailing(checked = contentExampleChecked)
+                        )
+                        OdsChoiceChipsFlowRow(
+                            selectedChip = header,
+                            outlinedChips = true,
+                            modifier = Modifier.padding(horizontal = dimensionResource(id = R.dimen.screen_horizontal_margin)),
+                        ) {
+                            Subtitle(textRes = R.string.component_modal_drawer_header_image)
+                            OdsChoiceChip(textRes = R.string.component_element_avatar, value = ComponentNavigationDrawersContentState.HeaderImage.Avatar)
+                            OdsChoiceChip(
+                                textRes = R.string.component_modal_drawer_background,
+                                value = ComponentNavigationDrawersContentState.HeaderImage.Background
+                            )
+                            OdsChoiceChip(textRes = R.string.component_element_none, value = ComponentNavigationDrawersContentState.HeaderImage.None)
+                        }
+                        OdsListItem(
+                            text = stringResource(id = R.string.component_modal_drawer_subtitle),
+                            trailing = OdsSwitchTrailing(
+                                checked = subTitleChecked,
+                                enabled = isContentExampleChecked
+                            )
+                        )
+                        OdsListItem(
+                            text = stringResource(id = R.string.component_modal_drawer_list_icon),
+                            trailing = OdsSwitchTrailing(
+                                checked = listIconChecked,
+                                enabled = isContentExampleChecked
+                            ),
+                        )
+                        OdsChoiceChipsFlowRow(
+                            selectedChip = content,
+                            outlinedChips = true,
+                            modifier = Modifier.padding(horizontal = dimensionResource(id = R.dimen.screen_horizontal_margin)),
+                        ) {
+                            Subtitle(textRes = R.string.component_modal_drawer_list_example)
+                            OdsChoiceChip(
+                                textRes = R.string.component_element_divider,
+                                value = ComponentNavigationDrawersContentState.SectionListExample.Divider
+                            )
+                            OdsChoiceChip(
+                                textRes = R.string.component_element_label,
+                                value = ComponentNavigationDrawersContentState.SectionListExample.Label
+                            )
+                            OdsChoiceChip(textRes = R.string.component_element_none, value = ComponentNavigationDrawersContentState.SectionListExample.None)
```

Comment:
I think chips in "Section list example" should be disabled when "Content example" switch is off (same behaviour as "Subtitle" and "List icon" switches:

```kotlin
OdsChoiceChip(
    textRes = R.string.component_element_divider,
    value = ComponentNavigationDrawersContentState.SectionListExample.Divider,
    enabled = isContentExampleChecked
)
OdsChoiceChip(
    textRes = R.string.component_element_label,
    value = ComponentNavigationDrawersContentState.SectionListExample.Label,
    enabled = isContentExampleChecked
)
OdsChoiceChip(
    textRes = R.string.component_element_none,
    value = ComponentNavigationDrawersContentState.SectionListExample.None,
    enabled = isContentExampleChecked
```

We can also reset "Section list example" value to "None" when disablinh "Content example":

```kotlin
if (!isContentExampleChecked) {
    listIconChecked.value = false
    subTitleChecked.value = false
    content.value = ComponentNavigationDrawersContentState.SectionListExample.None
}
```